src/v2/Apps/Fair/routes.tsx

```diff
@@ -129,7 +131,7 @@ export const routes: RouteConfig[] = [
 function initializeVariablesWithFilterState(params, props) {
   const initialFilterState = props.location ? props.location.query : {}
 
-  let aggregations: string[] = ["TOTAL", "GALLERY", "MAJOR_PERIOD"]
+  let aggregations: string[] = ["TOTAL", "GALLERY", "MAJOR_PERIOD", "ARTIST"]
```

Comment:
Add the `ARTIST` aggregation to the upfront query (so we can render the list of artists with works at the fair).